### DISCUSSION TITLE
Can rtx prevent me from using `pip install` from outside a venv?

### DISCUSSION THREAD

**Discussion by baggiponte** [jdx/mise#983] [Q&A]
I am using rtx to manage every language/tool on my machine (awesome work, ggs and kudos!). Sometimes, though, I happen to call `pip install` from outside of a Python environment. Is there a way rtx can warn me if I am doing that?

This is tedious because if I pip install something I cannot really remove it with `pip uninstall` (dependencies of that project will not be uninstalled, lol) so I have to remove said Python version and re-install it.

**Comment by jylenhof** [comment]
It should be probably better that rtx behavior will be fix...
But your are probably looking for setting env variable PIP_REQUIRE_VIRTUALENV=1
